extensions-contrib/kubernetes-overlord-extensions/src/main/java/org/apache/druid/k8s/overlord/KubernetesTaskRunner.java

```diff
@@ -256,6 +256,7 @@ JobResponse monitorJob(Pod peonPod, K8sTaskId k8sTaskId)
   @Override
   public void updateStatus(Task task, TaskStatus status)
   {
+    log.info("Updating status %s", status);
```

Comment:
this log line doesn't really include any task-specific info. can you add that too?